lib/options.dart

```diff
@@ -4,39 +4,65 @@ import 'dart:ui';
 import 'package:image/image.dart' hide Color;
 import 'package:iproov_flutter/enums.dart';
 
+// TODO: For all the classes in this file, identify which properties need to be null
+// and which ones don't. Non-nullable should be the default.
+// Note: non-nullable variables need to be required or have a default value in the constructor.
+
 class Options {
-  UiOptions ui = new UiOptions();
-  NetworkOptions network = new NetworkOptions();
-  CaptureOptions capture = new CaptureOptions();
+  const Options({
+    this.ui = const UiOptions(),
+    this.network = const NetworkOptions(),
+    this.capture = const CaptureOptions(),
+  });
+  final UiOptions ui;
+  final NetworkOptions network;
+  final CaptureOptions capture;
```

Comment:
I made all properties `final` and added `const` constructors. This makes all the options **immutable**/**read-only**. See example app for how to use them.